benchexec/tools/metaval.py

```diff
@@ -108,14 +111,22 @@ def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
                 ).Tool()
 
         if verifierName in self.wrappedTools:
```

Comment:
Just noticed now (but it actually had the same behavior before):
There is no way this condition can evaluate to `False`, except if a concurrent thread would delete something from `self.wrappedTools`, but of course this won't happen. So we could remove this condition and have less indentation.

Reply:
I would rather get this PR done and not misuse it for forcing me to do refactorings. If there are concerns about the code quality please just open an issue.

>  except if a concurrent thread would delete something

exactly this, so let's be defensive and not change this.

Reply:
There is no existing code that removes stuff from `self.wrappedTools`, so no need to be defensive here. Other threads randomingly changing stuff from your fields would break a lot anyway.

Reply:
I have no nerves to discuss this, so I will just remove it